gnmvidispine/vs_item.py

```diff
@@ -1115,6 +1115,16 @@ def commit(self):
         :return: None
         """
         path="{0}/metadata".format(self.parent.path())
+        test_for_str = False
+        try:
+            if not isinstance(self.as_xml(encoding="utf8").decode("utf8"), unicode):
+                raise TypeError("Argument does not have type 'unicode'")
+        except:
```

Comment:
would't it be more concise to simply:

```
if not isinstance(self.as_xml(encoding="utf8").decode("utf8"), unicode):
    if not isinstance(self.as_xml(encoding="utf8").decode("utf8"), str):
       raise TypeError("Argument does not have type 'str'")
```

no point catching and re-raising exceptions here surely?

Reply:
I tried something like that but it does not work under Python 3. 'unicode' is not valid in Python 3 hence the need to catch the exception then run the code for Python 3.